docker/jenkins/config.dict

```diff
@@ -0,0 +1,14 @@
+{
+  'image': 'dalg24/dtk-stack',
+  'tag': 'latest',
+  'name': os.getenv("BUILD_TAG"),
```

Comment:
no the `import os` is in the Python script `/usr/local/bin/docker_run.py` that is in the Docker slave container